README.md

```diff
@@ -0,0 +1,83 @@
+# NVA data model
+
+This is version 2 of the data model, as it entirely differs from the approach 
+taken in [version 1](https://github.com/BIBSYSDEV/nva-datamodel-java), the 
+decision was taken to separate the code entirely.
+
+The data model comprises the following components:
+-   Data Transfer Objects (DTOs) (TODO)
+
+-   Data Access Objects (DAOs) (TODO)
+
+-   Client APIs
+    -   Java client (TODO)
+    -   JavaScript client (TODO)
+
+-   Ontological representations for:
+    -   DTOs (TODO)
+    -   DAOs (TODO)
+    -   JSON-LD contexts for the DTOs (TODO)
+
+## Working with the data model
+
+The main point-of-reference for users of the data model is the DTOs; the client
+APIs simplify working with these.
+
+The clients can be used via maven/gradle (Java) and npm (JavaScript):
+-   gradle: ```shell TODO```
```

Comment:
I'll approve, but did you mean to finish these TODOs in this PR?

Reply:
No, they need doing at a point when I know what I am going to write here :)